tests/utils/compression.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import Path
+import logging
+
+import tfs
+from omc3.model.constants import TWISS_ELEMENTS_DAT, TWISS_DAT, TWISS_ADT_DAT, TWISS_AC_DAT
+
+LOGGER = logging.getLogger(__name__)
+
+def file_compression(file_path: Path, compress: bool = True) -> None:
+    """Compress or decompress a file. 
+    If compress is True, the file will be compressed, otherwise it will be decompressed.
+    Will only compress if the file exists and emits a warning to the logger if the file does not exist.
+
+    Args:
+        file_path (Path): The file to compress or decompress.
+        compress (bool, optional): Whether to compress the file. Defaults to True.
+    """
+    read_path  = file_path.with_suffix(".bz2") if not compress else file_path
+    write_path = file_path.with_suffix(".bz2") if     compress else file_path
+    if read_path.exists():
+        data = tfs.read(read_path)
+        tfs.write(write_path, data)
+    else:
+        LOGGER.warning(f"File {read_path} does not exist. Cannot {'' if compress else 'de'}compress.")
```

Comment:
I would like the function name and docstring to explicitely state we're talking about TFS files here.